.github/workflows/ci.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   linked_issue:
     name: Jira
     runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
```

Comment:
If this job isn't supposed to run on pushes to `main`, we can skip it.

This will reduce noise and allow contributors and maintainers to focus on failures that are actually problematic.